### DISCUSSION TITLE
Stucked on "Lesson 1: Duet Exercises: Setup" (part 13)

### DISCUSSION THREAD

**Discussion by karpatit** [OpenMined/courses#272] [General]
Hi, 
First, thank you for your new course on Federated Learning.
I get to the 13th part of the first lesson, followed the instructions, and get my environment ready to work. Then I wanted to continue to the next slide, but the button is not enabled. I refreshed many times the page, get out, log in again... but nothing helps. How can I get to the next part?

**Comment by tcp** [comment]
Hi @karpatit! I'll take a look at this. In the meantime, can you log this error [here](https://github.com/OpenMined/openmined/issues)? Please let us know what device you are using. Thank you!

**Reply by nigoshh** [reply]
I have the same problem, I will open an issue for this.

**Reply by nigoshh** [reply]
Quick fix for @karpatit: try lowering your browser's zoom level. A similar thing just happened to me on another site: there wasn't any confirmation button, and it appeared only when I decreased the zoom level.

**Reply by yemikifouly** [reply]
@nigoshh Thank you for the quick fix. 🙌 
@karpatit Have you tried this fix? Did it fix the issue for the time being?

**Reply by karpatit** [reply]
Hi, thank you for the response. Indeed, changing the zoom level helped!
